controllers/auth.controller.js

```diff
@@ -0,0 +1,53 @@
+const authService = require("../services/auth.service");
+require("dotenv").config();
+
+const registerUser = async (req, res) => {
+
+  if (JSON.stringify(req.body)==="{}") {
+    return res.send({ status: 400, message: "Request body is empty" });
+  }
+  try {
+    
+    const response = await authService.registerUser(req.body);
+
+
+    if (response.status==201) {
+      res.cookie("jsontoken", response.message, { httpOnly: true });
+      response.message = "Registration is successful";
+    
+    } 
+    res.send(response);
+    
+  } catch (err) {
+    if (err.name === "SequelizeUniqueConstraintError") {
+      res.send({ status: 409, message: err.parent.sqlMessage });
```

Comment:
does the client need to know about internal database error?

Reply:
This err.parent.sqlMessage shows **"message": "Duplicate entry 'rakib' for key 'username'",  "message": "Duplicate entry 'rakib@cefalo.com' for key 'email'"** , this types of duplicate error message